lms/services/lti_outcomes.py

```diff
@@ -137,19 +137,7 @@ def _get_body(cls, data):
             raise ExternalRequestError("Malformed LTI outcome response") from err
 
         if status != "success":
-            description = None
-            try:
-                # Look for an imsx_description to pass along to the client, but it is possible this field
-                # may not exist.
-                description = header["imsx_POXResponseHeaderInfo"]["imsx_statusInfo"][
-                    "imsx_description"
-                ]
-            except KeyError:
-                pass
-
-            raise ExternalRequestError(
-                message="LTI outcome request failed", extra_details=description
-            )
```

Comment:
This bit of code was putting part of the LTI outcomes API response body into `extra_details` as an attempt to debug something. See <https://github.com/hypothesis/lms/pull/2575>. Since we now report the entire response body to Sentry and Papertrail for all external requests, this is no longer needed.